models/Event.js

```diff
@@ -14,15 +14,15 @@ const EventSchema = new mongoose.Schema({
     require: true,
   },
   startTime: {
-    type: Number,
+    type: String,
     require: true,
   },
   recurringDates: { //this is not going to be correct, need to discuss with InstinctDev on how to format to match her input form.
     type: String,
     required: true,
   },
-   endDate: {
-    type: Number,
+   endTime: {
```

Comment:
I know the form had an endTime, but that wasn't in the schema. endDate was intended to be a cut-off for events to stop showing in views. For example, Huntober banki sessions would end on 10/31/22. 

Also, it's my opinion that we do not need an end time. Events end whenever, and I don't think it's entirely relevant to input or keep track of. What are your thoughts?

Reply:
Ah got it! In that case we can change it back/remove end time from the form